platform/kobo/koreader.sh

```diff
@@ -80,6 +80,9 @@ if [ ! -n "${PLATFORM}" ] ; then
 fi
 # end of value check of PLATFORM
 
+# Remount SD to RW, ignore errors since we may not have sd card
```

Comment:
Instead of ignoring errors wouldn't it be better to do a test like `if [ -f /mnt/sd ]; then` or `mount | grep /tmp/sd` or something?